src/containers/IndexCreature.js

```diff
@@ -1,10 +1,29 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
+
+import fetchData from "../functions/fetchData";
+import AllCreaturesTile from '../components/AllCreaturesTile'
 
 const IndexCreature = props => {
+  const [creatures, setCreatures] = useState([]);
+
+  const apiEndpoint = `/api/v1/adoptable/1` // hardcoded - update later
```

Comment:
Please update route to be dynamic. I see that you have it slated for later, however this feature is not complete until that has been done.